clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/ArraysTests.scala

```diff
@@ -413,8 +413,8 @@ trait ArraysTests { self: BaseIntegrationSpec =>
     val rootSource = rootTestStorageDir / s"arrays/$chipwellBarcode/v$version/"
     val vcfSource = rootSource / vcfName
     val vcfIndexSource = rootSource / vcfIndexName
-    val grnIdatSource = rootSource / grnIdatName
-    val redIdatSource = rootSource / redIdatName
+    val grnIdatSource = rootSource / "idats" / grnIdatName
```

Comment:
Why not use `DeliverArraysExecutor. IdatsDir` here?

Reply:
That constant ends with a `/`, wasn't sure if it'd mess things up when paired with the method `/`.